### ISSUE TITLE
onActionClick doesn't work

### ISSUE THREAD

**Issue by steplerbox** [indexiatech/re-notif#15]
Pass props to Notif in Notifs component:
https://github.com/indexiatech/re-notif/blob/master/src/components/Notifs.js#L16
<Notif
      key={getter(notif, 'id')}
      message={getter(notif, 'message')}
      kind={getter(notif, 'kind')}
      componentClassName={componentClassName}
      CustomComponent={CustomComponent}
      onActionClick={onActionClick}
      actionLabel={actionLabel}
/>

Check onClick in Notif component:
https://github.com/indexiatech/re-notif/blob/master/src/components/Notif.js#L16
 if (this.props.onClick) {

How onClick can appear in Notif props?

**Comment by NameFILIP** [comment]
yeah, I think it should be 
```
if (this.props.onActionClick) {
  this.props.onActionClick();
```

**Comment by kylecesmat** [comment]
Fixed by #19 - going to close this issue
